geo/src/main/java/org/odk/collect/geo/GeoPointActivity.kt

```diff
@@ -10,13 +13,19 @@ import org.odk.collect.geo.analytics.AnalyticsEvents
 import org.odk.collect.strings.localization.LocalizedActivity
 import javax.inject.Inject
 
-class GeoPointActivityNew : LocalizedActivity(), GeoPointDialogFragment.Listener {
+class GeoPointActivity : LocalizedActivity(), GeoPointDialogFragment.Listener {
 
     @Inject
     lateinit var geoPointViewModelFactory: GeoPointViewModelFactory
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
It seems like we can't do this on older versions (without using reflection to call private methods). Alternative methods involve messing around with themes and I think that'll add more complexity than it's worth.